README.md

```diff
@@ -0,0 +1,44 @@
+# NodeScript SQL Adapter
+
+Facilitates SQL nodes in [NodeScript](https://nodescript.dev).
+
+## Development
+
+### Requirements
+
+- Node
+- Postgres
+- MySql
+
+### Run locally:
+
+Install with `npm i` then run with `npm run dev`. Ensure you have a supported sql server running with a database configured.
+
+Make requests to `http://localhost:8183/Sql/<endpoint>`. Refer to the schemas in `/protocol` for further information on the requirements for each request.
+
+
+## Configuration
+
+NodeScript Sql Adapter can be configured with the following environment variables:
+
+- **AUTH_SECRET** — authentication shared secret, used to restrict access from graphs to the adapter.
+
+- **POOL_SIZE** (default: 5) - the maximum number of connections to establish to *each* database the adapter connects to.
+
+- **POOL_TTL_MS** (default: 60_00) - pools created that many millis ago will be recycled (this eliminates connection leaks otherwise occurring with high-throughput scenarios)
```

Comment:
Small typo here i think with the `60_00` ? I guess it should be `60_000`?

Reply:
oh good catch, thanks!